spec/person_spec.rb

```diff
@@ -0,0 +1,36 @@
+require './person'
+require './book'
+require './rental'
+
+describe Person do
+  let(:person) { Person.new(31, 'Enoh Collins') }
+
+  it 'should initialize correctly' do
+    expect(person.age).to eq(31)
+    expect(person.name).to eq('Enoh Collins')
+  end
+
+  it 'should correctly check age' do
+    expect(person.send(:of_age?)).to be true
+    person = Person.new(15, 'AbdulSami')
+    expect(person.send(:of_age?)).to be false
+  end
+
+  it 'should return correct name' do
+    expect(person.correct_name).to eq('Enoh Collins')
+  end
+
+  def add_rental(book, due_date)
+    rental = Rental.new(book, self, due_date)
+    @rentals << rental
+    rental
+  end
```

Comment:
- [ ] Well I can understand that your previous code `reviewer` asked for adding the `test` case for the missing `methods` like this `add_rental` method. But it seems like instead of adding the `test` cases for checking the `method` functionality correctly, you again implemented the `method` for the sake of milestone requirements fulfilment, which is totally unacceptable way, it is just like `re-inventing` the wheel or kind of `useless` efforts. Make sure, you should write the test for checking the `add_rental` method for the `person` class, for better understanding, please take a look at your `person.rb` and see the methods you implemented over there. Be consistent and focus on the reviewer comments in order to pass this milestone.